docs/pages/build-reference/infrastructure.mdx

```diff
@@ -31,9 +31,9 @@ When selecting an image for the build you can use the full name provided below o
 
 Android builders run on virtual machines in an isolated environment. Every build gets its own dedicated VM instance.
 
-- Build resources:
-  - [`medium`](eas-json/#resourceclass-1): 4 CPU, 16 GB RAM ([`n2-standard-4`](https://cloud.google.com/compute/docs/general-purpose-machines#n2_machines) Google Cloud machine type)
-  - [`large`](eas-json/#resourceclass-1): 8 CPU, 32 GB RAM ([`n2-standard-8`](https://cloud.google.com/compute/docs/general-purpose-machines#n2_machines) Google Cloud machine type)
+- Hardware:
+  - 4 CPU, 16 GB RAM (n2-standard-4 Google Cloud machine type)
+  - 8 CPU, 32 GB RAM (n2-standard-8 Google Cloud machine type)
```

Comment:
Link to pricing page descriptions from here https://github.com/expo/universe/pull/12408